autogen/main/variables.tf.tmpl

```diff
@@ -301,6 +301,14 @@ variable "configure_ip_masq" {
   default     = false
 }
 
+{% if beta_cluster %}
+variable "cluster_telemetry_type" {
+  type        = string
+  description = "Available options include ENABLED, DISABLED, and SYSTEM_ONLY"
+  default     = "ENABLED"
```

Comment:
The default should probably be null actually (so we don't set this block at all).

Reply:
Done.